lib/widgets/contact_tile.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+class ContactTile extends StatelessWidget {
+  final String firstName;
+  final String lastName;
+  final String phoneNumber;
```

Comment:
Can a variable declared with final be modified? If  NO, then consider not using final. Contacts will be modified.